tq/transfer.go

```diff
@@ -33,11 +33,21 @@ type Transfer struct {
 	Oid           string       `json:"oid,omitempty"`
 	Size          int64        `json:"size"`
 	Authenticated bool         `json:"authenticated,omitempty"`
-	Actions       ActionSet    `json:"actions,omitempty"`
+	Actions       mapActionSet `json:"actions,omitempty"`
+	Links         mapActionSet `json:"_links,omitempty"`
 	Error         *ObjectError `json:"error,omitempty"`
 	Path          string       `json:"path,omitempty"`
 }
 
+// HybridActions returns an action set composed of the "actions" property, as
+// well as the "_links" property. It is implemented by a multiActionSet, and
+// therefore retains that ordering.
+func (t *Transfer) HybridActions() ActionSet {
```

Comment:
_Great_ call. I suppose the general rule is that composing single-functioned interfaces together can usually be accomplished by another function, instead of another interface. That all being said, I found that we already had `Rel(name string) (a *Action, ok bool)`, which I re-purposed to be `Rel(name string) (a *Action, err error)` and take on this functionality in 3180a2d.